backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -521,21 +526,22 @@ func (ctx *AppContext) processNewTickets(nt NewTicketsForBlock) {
 
 		msgTx, err := MsgTxFromHex(n.hex)
 		if err != nil {
-			log.Warnf("MsgTxFromHex failed for %v: %v", n.hex, err)
+			log.Warnf("processNewTickets: MsgTxFromHex failed for %v: %v", n.hex, err)
```

Comment:
If we end up adding an error package like suggested in #518 , we might also start using the convention in dcrwallet where the operation is declared at the beginning of a function:
const op = "processNewTickets"
Then when we make an error
err := fmt.Errorf("%v: everybody died", op)
Just a suggestion tho, we can always decide later when/if that issue gets worked on.

Reply:
We can go through and do this later in #518. I might actually make a start on that this afternoon